character-sfx.js

```diff
@@ -86,24 +86,25 @@ class CharacterSfx {
     this.oldNarutoRunSound = null;
     this.lastEmote = null;
 
-    if (this.player.isLocalPlayer) {
-      const wearupdate = e => {
-        sounds.playSoundName(e.wear ? 'itemEquip' : 'itemUnequip');
-      };
-      player.addEventListener('wearupdate', wearupdate);
-      this.cleanup = () => {
-        player.removeEventListener('wearupdate', wearupdate);
-      };
-    }
-
     this.currentStep = null;
     this.currentSwimmingHand = null;
     this.setSwimmingHand = true;
+    this.setEquipSound = false;
   }
   update(timestamp, timeDiffS) {
     if (!this.player.avatar) {
       return;
     }
+    if (this.player.isLocalPlayer && !this.setEquipSound) {

```

Comment:
If this is a listener then it should not be listened in the main loop. This belongs in setup code.

Reply:
https://github.com/webaverse/app/pull/3545/files

create characterSfx for localPlayer after `isLocalPlayer` has been set (like characterPhysics)